src/components/DateParagraph/DateParagraph.styles.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const StyledDateParagraph = styled.p`
+	font-weight: 400;
+	color: #969696;
```

Comment:
Kolor Ci się powiela - spróbuj zrobić Theme Providera od styled components, umieść sobie to w zmiennej i wykorzystaj zmienną z theme. Wymagam więcej bo ogarniałeś najbardziej :)